### PR TITLE
Changelog fragment copy edits.

### PR DESCRIPTION
**Pull request by dharmabumstead**
##### SUMMARY

Changelog fragment copy edits.

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: changelogs/fragments/display_fix.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 bugfixes:
-- ensure displayed messages under peristent connections are returned to the controller (https://github.com/ansible/ansible/pull/36064)
+- updetd to ensure displayed messages under peristent connections are returned to the controller (https://github.com/ansible/ansible/pull/36064)
```
Comment (nitzmahone) [inline changelogs/fragments/display_fix.yaml]: updated

File: changelogs/fragments/v2.5.0b1_catchup.yaml [hunk 2]
```diff
@@ -129,7 +133,7 @@ new_connection_plugins:
 - netconf
 - |
   network\_cli
-   - While neither network\_cli nor netconf are technically new plugins, these connections may now be used directly with network modules. See
+   - The existing network\_cli nor netconf plugins connections can now be used directly with network modules. See
```
Comment (nitzmahone) [inline changelogs/fragments/v2.5.0b1_catchup.yaml]: s/nor/and/

**Review by nitzmahone — CHANGES_REQUESTED** [review]
just a couple minor edits needed

File: changelogs/fragments/v2.5.0b1_catchup.yaml (hunk 2 above)
Comment (nitzmahone) [reply changelogs/fragments/v2.5.0b1_catchup.yaml]: s/plugins connections/connection plugins/

Comment (dharmabumstead) [reply changelogs/fragments/v2.5.0b1_catchup.yaml]: Straggler removed.

File: changelogs/fragments/display_fix.yaml (hunk 1 above)
Comment (dharmabumstead) [reply changelogs/fragments/display_fix.yaml]: Oops. Fixed.

**Pull request merged**